web/webkit/src/main/scala/net/liftweb/http/HtmlNormalizer.scala

```diff
@@ -0,0 +1,291 @@
+package net.liftweb
+package http
```

Comment:
(Which isn't to say you're wrong, just saying that's a nice byproduct and as we consider the issue you've brought up, it's worth keeping that aspect in mind.)